Yona/Yona/Others/Constants.swift

```diff
@@ -132,8 +132,15 @@ struct YonaConstants {
     struct nsUserDefaultsKeys{
```

Comment:
i think these are ok as Structs, but when you have a max number of types make enum